src/App.js

```diff
@@ -1,9 +1,53 @@
+/* eslint-disable import/extensions */
+/*
+reason for this is that I did try to change it several times
+and it gave me contradicting rules
+namely:
+- Missing file extension for "./components/Calculator"  import/extensions
+- Unexpected use of file extension "js" for "./components/home.js"   import/extensions
+- Missing file extension for "./components/home"
+*/
+import {
+  BrowserRouter, Routes, Route, Outlet,
+} from 'react-router-dom';
 import './App.css';
 import Calculator from './components/calculator';
+import Home from './components/home';
+import Quote from './components/quote';
+
```

Comment:
- [ ] Please fix the spelling of the importing components where they should start with capital, not with small letter
 
![image](https://user-images.githubusercontent.com/74744787/221232423-5f7d9ad7-91c0-44cb-90dc-0c5582f0b7b4.png)

Reply:
@MeqdamAlqudah, I did try to change the path several times, if the problem still exists, please inform me about it.